test/extended/router/scoped.go

```diff
@@ -174,18 +174,18 @@ var _ = g.Describe("[sig-network][Feature:Router][apigroup:route.openshift.io][a
 
 		g.It("should override the route host for overridden domains with a custom value [apigroup:image.openshift.io]", func() {
 
-			configPath := exutil.FixturePath("testdata", "router", "router-override-domains.yaml")
-			g.By(fmt.Sprintf("creating a router from a config file %q", configPath))
-			err := oc.AsAdmin().Run("new-app").Args("-f", configPath, "-p", "IMAGE="+routerImage).Execute()
+			routerPod := createOverrideDomainRouterPod(routerImage)
+			g.By("creating a router")
+			ns := oc.KubeFramework().Namespace.Name
+			_, err := oc.AdminKubeClient().CoreV1().Pods(ns).Create(context.Background(), routerPod, metav1.CreateOptions{})
 			o.Expect(err).NotTo(o.HaveOccurred())
 
-			ns := oc.KubeFramework().Namespace.Name
 			execPod := exutil.CreateExecPodOrFail(oc.AdminKubeClient(), ns, "execpod")
 			defer func() {
 				oc.AdminKubeClient().CoreV1().Pods(ns).Delete(context.Background(), execPod.Name, *metav1.NewDeleteOptions(1))
 			}()
 
-			g.By(fmt.Sprintf("creating a scoped router with overridden domains from a config file %q", configPath))
+			g.By("creating a scoped router with overridden domains")
```

Comment:
This `g.By` seems superfluous anyway; I wouldn't mind if you deleted it or changed it to something like `g.By("waiting for the router pod to become ready")`.